packages/headless/src/controllers/commerce/product-listing/result-list/headless-product-listing-interactive-result.ts

```diff
@@ -46,12 +44,11 @@ export function buildInteractiveResult(
       return;
     }
     wasOpened = true;
-    engine.dispatch(logProductRecommendationOpen(props.options.result));
+    // TODO: Log on click
```

Comment:
Same as above, this event is not kosher AFAIK.

Reply:
Add a ticket 👍

Reply:
@samisayegh Do you have an epic/ticket of some sort for analytics for new commerce engine ?

Reply:
Yep, we've got [LENS-1595](https://coveord.atlassian.net/browse/LENS-1595) specifically for clicks.